pkg/services/ngalert/api/authorization_test.go

```diff
@@ -49,7 +49,7 @@ func TestAuthorize(t *testing.T) {
 		}
 		paths[p] = methods
 	}
-	require.Len(t, paths, 48)
+	require.Len(t, paths, 49)
```

Comment:
Magic number? Do we want to extract in to a separate var and add code comments?

Reply:
It's a strange test, in my experience it's here basically to remind you to add an entry in authorization.go if you've forgotten. It's reminded me a couple times, so I think it's doing something at least.

Reply:
It makes sure that all paths are covered by authorization :)